mysql-test/suite/rpl/t/rpl_percona_userstat.test

```diff
@@ -0,0 +1,60 @@
+--echo #
+--echo # Bug 995624 RBR events are not reflected in userstat's
+--echo # Rows_updated
+--echo #
+
+--source include/master-slave.inc
+
+connection slave;
+set @slave_userstat_saved= @@global.userstat;
+set global userstat=ON;
+--echo Rows_updated on slave - starting situation
+select ROWS_UPDATED from information_schema.client_statistics;
+
+connection master;
+set @master_userstat_saved= @@global.userstat;
+set global userstat=ON;
+--echo Rows_updated on master - starting situation
+select ROWS_UPDATED from information_schema.client_statistics;
+
+create table t1 (m int);
```

Comment:
Although not required, making SQL keywords uppercase makes the code more readable.
For instance, I would write the following
```sql
CREATE TABLE t1 (m INT);
--echo 3 "updates" to rows
INSERT INTO t1 VALUES(15), (16), (17); 
--echo 1 "update" to rows (4 in total)
UPDATE t1 SET m = 20 WHERE m = 16;
--echo 1 "update" to rows (5 in total)
DELETE FROM t1 WHERE m = 17;
```